runtime/testy/namedSensors.csv

```diff
@@ -0,0 +1,3 @@
+two,2
```

Comment:
FYI last year we decided that peripheral names must match the regex `^[A-Za-z][A-Za-z0-9]*$` (i.e. alphanumeric, but can't start with a number). Hopefully the code for this checking is still a part of Dawn, and runtime doesn't have to do anything about this. (I'm bringing this up because serialization schemes may break if you allow arbitrary character sets.)